### DISCUSSION TITLE
OpenAI抓作弊

### DISCUSSION THREAD

**Discussion by Clare0808** [1101-GS4719-C/Python-Community-GS4719-C#2034] [General]
https://technews.tw/2024/08/06/openai-says-its-taking-a-deliberate-approach-to-releasing-tools-that-can-detect-writing-from-chatgpt/

現在真的好多人都會用ChatGPT來寫文章，不管是平時老師們派的作業，還是爭取大學資格所繳交的檔案。那些不知道如何下筆的人，總是靠ChatGPT來完成。這點真的非常的不公平，如果審閱的人有看出來，並且不給予分數，那到沒有問題。現在就怕當下沒有發現文章不是本人親寫卻得到高分的人……

不過還好現在有OpenAI可以幫忙，它可以偵測文章中，是否有因使用ChatGPT生成而產生的浮水印，而且準確率極高。但OpenAI在判斷時還是會有一些小問題，可能要待日後的改善才能夠正式啟用。

希望這個功能以後可以大大推行，告別那些不公正的評分。

**Comment by peresonacifer** [comment]
我十分認同你的觀點，現在的ai能做到非常多的事，尤其在學術方面，學生不應該濫用ChatGPT來完成他們的報告和作業，因為不只成績有失公正性，甚至根本毫無參考價值，ChatGPT應該是用來輔助學習的，所以防作弊機制是一定需要的。

**Comment by ds0w0** [comment]
我自己偶爾也會用GPTZero來檢查文章，我認為作為檢測AI的準確率還蠻高的，至於學生使用ChatGPT我認為能夠懂得利用這項工具才能夠增加自己的生產力，AI的效率勢必比人類高，但人類的想法才是最為關鍵的，我認為凡使用AI而不經修改校稿是容易辨認的，善用AI來加速工作進程不可否認已成為現在的日常。

**Comment by william-60760** [comment]
我覺得文字浮水印的偵測方式有機會在未來成為主要的防作弊機制。在這個技術爆炸的時代，未來AI所生成的文字將會越來越接近人們親自書寫出來的文字。到時候判斷文章的用字遣詞將不再百分百有效，反而是利用不相關詞語間暗含的浮水印判斷將更為直接有效，不容易像用字有模糊的判斷空間。

**Comment by Nana1218** [comment]
我跟你的想法很相似，但我有一點認為用AI去協助寫作業應該會是未來的大方向，因為現在有許多人自學就是使用AI，那人類在未來應該會是更依賴AI的。

**Comment by kingsophy** [comment]
在未來AI一定會蓬勃發展，因此我認為防作弊機制是必要的，一方面是讓認真的人能夠得到該有的收穫，另一方面是讓那些用小聰明的人不能再靠AI來得到不屬於自己的報酬。

**Comment by hunteryang0108** [comment]
有些作業確實不應該讓AI代寫，這些作業應該要嚴格查核是否有代寫作弊的情況發生，因此有這個抓作弊工具的存在式非常有必要的。但是對於那些重複性高的報告或文件，我認為不應該在使用生成式AI方面加以限制，因為學生應該把時間花在其他更有意義，更值得思考的領域。

**Comment by CtyceT** [comment]
新的發明的出現需要伴隨著各類的制度去規範。生成式AI非常年輕，但依賴它的人愈來愈多，因此各類的審核與防弊的機制相對也非常重要。為了維持實質上的公平，伴隨而來的不同觀念糾正和防弊機制的創造也很值得去深入思考。

**Comment by Jerryguo888** [comment]
我認為chat GPT 是一種輔助工具，可以在我們毫無頭緒時，提供一些想法，但不代表可以直接叫chat GPT來幫你寫作業。為了維持公平性，我認為我們應該需要制定防作弊機制，減少作弊事件。

**Comment by lai1008** [comment]
我認為ChatGpt只是一個輔助工具，可以從中幫助我們的學習，也可以在我們遇到困難的時候給予一些幫助，但若只有一昧的抄襲而不理解它的意思，這樣就對學習一點意義都沒有。

**Comment by qiu0109** [comment]
我覺得Chatgpt用來輔助寫作業是種趨勢，我自己使用下來也覺得gpt讓我節省了很多時間，我會將自己寫好的文章丟給gpt修飾，或是讓gpt翻譯與解釋講義上的內容，這會幫助我更快了解課程內容。但如果說有人用gpt直接寫報告而沒有嘗試去理解的話，我會認為這對學習一點幫助都沒有。

**Comment by wcyeee** [comment]
我也覺得現在應該要慢慢可以接受AI的輔助嗎？現在AI已經慢慢成為我們生活中不可或缺的一部分了，我覺得作業可以使用AI，但前提是只能是輔助，不要整個像貼文說的一樣交給AI寫。但我覺得也是個人的選擇？雖然AI方便快速，但我們一般人使用的AI水準不見得比人寫的還好，更符合作業需求，並且那些全部給AI寫的人也就少了許多學習的機會，變得過度依賴AI。我覺得最好的就是自己寫作業，然後可以用AI幫我檢查錯誤或是什麼邏輯問題之類的，這樣的使用感覺是目前比較好的使用方法。

**Comment by yuan3x2** [comment]
如何判定學生是否使用AI工具輔助裝撰寫，甚至直接套用生成內容確實是令老師頭痛的一個問題，然而現在還面臨第二種情況，有些工具判定是否為AI撰寫時並不是芬常準確，有些人完全未使用AI輔助完成文章也被判定有70%是由AI完成，還挺令人哭笑不得。

**Comment by 113607514** [comment]
我時常使用 GPTZero 來檢查文章，認為它作為檢測 AI 的工具，準確率相當高。至於學生使用 ChatGPT，我認為能善加利用這項工具，確實能提升生產力。AI 的效率無疑高於人類，但真正關鍵的還是人類的創意與想法。不過，完全依賴 AI 而不進行修改與校對的文章，往往很容易被識別。善用 AI 來加速工作進程，已經成為現代生活的一部分，也是一種不可忽視的趨勢。

**Comment by ed112601513** [comment]
我認為還是有一些可以鑽漏洞的空間，像是如果使用ChatGPT生成文章，再使用其他AI(如Copilot)來改寫文章或自己修改文章，應該也有機會避免被發現是AI生成的，而現在ChatGPT-4的強大有目共睹，所以我覺得這個時代的學習成績幾乎可以用pay to win來形容(買ChatGPT的付費方案就能獲得比較高的作業成績等)。

**Comment by Taoyi-Chiang** [comment]
昨天看到一個笑話，說AI發明是為了讓人類省下打掃洗衣煮飯的時間，可以去畫畫玩音樂做創作。
結果AI出來了，人類還是在打掃洗衣煮飯。
